CONTRIBUTING.md

```diff
@@ -97,7 +130,7 @@ brew install postgresql
 
 ### Installation
 
-First make sure that you set up your `virtualenv` as described in section _Setting up an environment_. Next, install dbt (and it's dependencies) with:
+First make sure that you set up your `virtualenv` as described in section _Setting up an environment_. Next, install dbt (and its dependencies) with:
```

Comment:
sorry

Reply:
[wasn't you](https://github.com/fishtown-analytics/dbt/commit/69a65dd63f396db3ce3452382cdd37f88ad59539#diff-6a3371457528722a734f3c51d9238c13)